stumpy/snippets.py

```diff
@@ -96,6 +96,7 @@ def _get_all_profiles(
         percentage = np.clip(percentage, 0.0, 1.0)
         s = min(math.ceil(percentage * m), m)
 
+    T_subseq_isconstant = core.rolling_isconstant(T, s)
     M_T, Σ_T = core.compute_mean_std(T, s)
```

Comment:
To be on the safe side, I think we should do:

```
T[np.isinf(T)] = np.nan
T_subseq_isconstant = core.rolling_isconstant(T, s)
M_T, Σ_T = core.compute_mean_std(T, s)
```

Right?

Reply:
> Right?

Why not call `core.prepocess` here?

> can we add T[np.isinf(T)] = np.nan to the core._preprocess(T)

That doesn't feel right. `core._preprocess` doesn't actually change the values of `T` (i.e., it only ensures that you get a copy that is returned, values unchanged, as a numpy array) and I would prefer to keep it that way

Reply:
> > Right?
> 
> Why not call `core.prepocess` here?

Right!!! Will revise the code.

Reply:
Did you forget to revise this?